CONTRIBUTING.md

```diff
@@ -14,26 +38,32 @@ Four spaces. No tabs, please.
 
 ### Linting
 
-We run all our our Python code (excluding migrations and settings)
+We run all of our Python code (excluding migrations and settings)
 through [flake8](https://flake8.readthedocs.org/en/2.2.3/). If you are
-using the vagrant-based development setup, there is a
-[fabric](http://www.fabfile.org/) task that will run flake8 with the
-correct settings.
+using the vagrant-based development setup, you can use this command to
```

Comment:
I think this needs to be "If you are using `otm-vagrant`..." and we need to figure out the appropriate command. (And likewise for the other commands below.) Only internal people will be able to `vagrant ssh app`.